packages/components/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 - Fixed display of reset button when using RangeControl `allowReset` prop.
 
+### Breaking Change
+
+- `ServerSideRender` is no longer part of components. It was extracted to an independent package `@wordpress/server-side-render`.
```

Comment:
I don't think we can fully deprecate it in WordPress context. It's fine for npm but inside WP-Admin it still should be possible to use `wp.components. ServerSideRender`.